apps/web/src/components/Dropzone/index.tsx

```diff
@@ -1,22 +1,25 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Image from 'next/image';
 import { DropzoneOptions, useDropzone } from 'react-dropzone';
 
-import { Container } from './styles';
+import { Container, Overlay } from './styles';
 
 interface DropzoneProps extends DropzoneOptions {
   currentFile?: any;
   label?: string;
   withPreview?: boolean;
-  children?: React.ReactNode;
+  children?: JSX.Element | JSX.Element[];
+  dropzoneRef?: React.RefObject<HTMLButtonElement | null>;
 }
 
 function Dropzone({
```

Comment:
Please leave this component the way it was before, create a new one called `AvatarDropzone`.

Reply:
done